scripts/pi-hole/js/login.js

```diff
@@ -112,15 +112,25 @@ $(function () {
   }).done(function (data) {
     if (data.session.valid === true) redirect();
     if (data.session.totp === true) $("#totp_input").removeClass("hidden");
+  });
+
+  // Get imformation about HTTPS port and DNS status
```

Comment:
Before Spellcheck complains...

```suggestion
  // Get information about HTTPS port and DNS status
```